src/Http/Role/GetPermissionsRequestHandler.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use CultuurNet\UDB3\Http\Response\JsonResponse;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use Fig\Http\Message\StatusCodeInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Headers;
+
+class GetPermissionsRequestHandler implements RequestHandlerInterface
+{
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $permissions = Permission::getAllPermissions();
+
+        $list = [];
+        foreach ($permissions as $permission) {
+            $list[] = $permission->toUpperCaseString();
+        }
+
+        return new JsonResponse(
+            $list,
+            StatusCodeInterface::STATUS_OK,
+            new Headers(['Cache-Control' => 'private'])
```

Comment:
In previous refactors we removed this header, I don't think there is any added value in doing this.